ansible/roles/redis-dump-monitoring/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+redis_monitoring_user: analytics
+redis_monitoring_home: "/home/{{redis_monitoring_user}}"
+redis_home: "{{redis_monitoring_home}}/redis-stable"
+redis_dump_rdb_time_in_seconds: 900
+redis_dump_rdb_keys_to_save_state: 1
+redis_dump_rdb_log_file_location: "/var/log/redisdumprdbstate.log"
+redis_dump_rdb_log_file_log_rotate_backup: 14
+redis_dump_rdb_mail_sender: "monitorning@sunbirded.org"
```

Comment:
@SMYALTAMASH - Is the spelling of the mail sender correct? check the monitoring spelling

Reply:
I've  created a new variable to identify alerts specific to redis dump monitoring @SanthoshVasabhaktula and sender email name can be anything, do you want me to use the same name as existing monitoring sender default name?

Reply:
Monitoring Spelling is updated.

Reply:
> @SMYALTAMASH - Is the spelling of the mail sender correct? check the monitoring spelling



Monitoring Spelling is updated @SanthoshVasabhaktula .